rich/progress.py

```diff
@@ -357,6 +357,30 @@ def render(self, task: "Task") -> Text:
         return Text(str(remaining_delta), style="progress.remaining")
 
 
+class CondensedTimeColumn(ProgressColumn):
```

Comment:
The naming isn't clear. What is condensed time? How is it different from TimeRemaining or TimeElapsed? I know the answers, but I don't think it would be obvious to a user.

I do like the functionality. But if we add it I think it should be rolled in to the `TimeRemainingColumn` class so that the defaults continue to work as before, but with options for the new features.

Reply:
Yeah, I don't love `Condensed`, but I chose that thinking of "a shorter time column with more information". Maybe `Compact` would be clearer.

However, I'm happy to roll it into `TimeRemainingColumn`. How about:

```python
class TimeRemainingColumn(ProgressColumn):
    """Renders estimated time remaining.

    Args:
        compact (bool, optional): Render MM:SS when time remaining is less than an hour. Defaults to False.
        elapsed_when_finished (bool, optional): Render time elapsed when the task is finished. Defaults to False.
    """
```

Or, should `compact` (or another name) encompass both behaviors? Or ???

Reply:
Done!